config_example/keys.json

```diff
@@ -1,5 +1,7 @@
 {
-    "imgbb": "",
-    "tmdb": "",
-    "imgbly": false
+	"imgbb": "",
+	"tmdb": "",
+	"imgbly": false,
+	"imgbly_thumb_size": 640,
+	"automatic_exit": false
```

Comment:
Queste due opzioni sono da rimuovere.
- `imgbly_thumb_size` vale solo per Imgbly, e visto che non è possibile impostare la larghezza delle thumbnails per gli altri due hosting, tanto vale toglierla (anche perché Imgbly viene usato principalmente come ripiego in caso di errori/sforamento della dimensione delle immagini, quindi ha poco senso).
- `automatic_exit` è inutile e riguarda solo la GUI, se il releaser vuole chiudere la finestra al termine del processo può farlo benissimo manualmente. Al massimo si può aggiungere un checkbox da selezionare ogni volta, ma un'opzione nel file `keys.json` no...